workflow/realisations.py

```diff
@@ -22,6 +22,7 @@
 from schema import Schema
 from velocity_modelling.bounding_box import BoundingBox
 
+from IM import im_calculation
```

Comment:
Whats IM? Its not a package and I can't see a module with that name?

Reply:
I think its meant to be
```suggestion
from im_calculcation import IM
```

Reply:
nope, the IM package is the actual import we want.

Reply:
It isn't called im_calculation for historical reasons... I should've changed that in the IM calc repo rewrite but I didn't.